services/chatbot.js

```diff
@@ -0,0 +1,39 @@
+const utils = require('../utils');
+
+class Chatbot {
+  constructor (nick, client) {
+    this.nick = nick;
+    this.client = client;
+  }
+
+  sendMessage (from, to, message) {
+    if (to.charAt(0) === '#') {
+      this.client.sendMessage(to, message);
+    } else {
+      this.client.sendMessage(from, message)
+    }
+  }
+
+  addListeners () {
+    this.client.addJoinHandler((channel, nick) => {
+      this.client.sendMessage(channel, `Welcome to Publiclab, ${nick}!`)
```

Comment:
Here we may want the bot to tell the user how to ask for help, what the correct syntax is, so they don't have to guess!